filament/backend/src/opengl/OpenGLProgram.cpp

```diff
@@ -30,6 +30,7 @@
 #include <utils/Systrace.h>
 
 #include <array>
+#include <sstream>
```

Comment:
I really don't want to include <sstream> because it can bring in a lot of code. I just saw it's used in `Material.cpp`, but that's a mistake, I'll fix it.

Reply:
removed.